admin/proc-using-azure-auth-provider.adoc

```diff
@@ -0,0 +1,141 @@
+[id='proc-using-azure-auth-provider_{context}']
+= Using Microsoft Azure as an authentication provider in {product}
+
+The `core-plugin-api` package in {product-short} comes integrated with Microsoft Azure authentication provider, authenticating signing in using Azure OAuth.
+
+.Prerequisites
+* You have deployed {product-short} on AKS. For more information, see xref:proc-rhdh-deploy-aks_admin-rhdh[].
+
+.Procedure
+
+. In the Azure portal, navigate to the Microsoft Enterprise ID service and select *App Registrations*.
+. Click *New Registration* and complete the form following the provided instructions.
+. Once the application is created, note down the following:
++
+--
+** `clientId`: Application (client) ID, found under App *Registration -> Overview*.
+** `clientSecret`: Secret, found under *App Registration -> Certificates & secrets (create new if needed).
+** `tenantId`: Directory (tenant) ID, found under *App Registration -> Overview*.
+--
+
+. Ensure the following fragment is included in your {product-short} ConfigMap:
++
+--
+[source,yaml]
+----
+auth:
+  environment: production
+  providers:
+    microsoft:
+      production:
+        clientId: ${AZURE_CLIENT_ID}
+        clientSecret: ${AZURE_CLIENT_SECRET}
+        tenantId: ${AZURE_TENANT_ID}
+        domainHint: ${AZURE_TENANT_ID}
+        additionalScopes:
+          - Mail.Send
+----
+You can either create a new file or add it to an existing one.
+--
+
+. Apply the ConfigMap to your Kubernetes cluster:
++
+--
+[source,bash]
+----
+kubectl -n <your_namespace> apply -f <app-config>.yaml
+----
+--
+
+. Create or reuse an existing Secret containing Azure credentials and add the following fragment:
++
+--
+[source,yaml]
+----
+stringData:
+  AZURE_CLIENT_ID: <value-of-clientId>
+  AZURE_CLIENT_SECRET: <value-of-clientSecret>
+  AZURE_TENANT_ID: <value-of-tenantId>
+----
+--
+
+. Apply the secret to your Kubernetes cluster:
++
+--
+[source,bash]
+----
+kubectl -n <your_namespace> apply -f <azure-secrets>.yaml
+----
+--
+
+Helm deployment::
```

Comment:
Is this intentional? Not sure why **_Helm deployment_** is between steps 7 and 8. I think we should remove this or incorporate it differently.

Reply:
Now that I see the **_Operator-backed deployment_** heading also, I think I see what we might be trying to do. These are different procedures and should ideally be their own proc modules. If you do want to leave the headings for now, let's format this differently. There are multiple ways, but one option is to restart the numbering at 1 under each heading. It would also be a good idea to add use more descriptive headings since _Helm deployment_ occurs multiple times in the docs and was already used as a heading elsewhere.

Reply:
separated in two headings.